laravel/resources/views/schedules/resident/schedule_confirm.blade.php

```diff
@@ -86,10 +103,21 @@
 		</p>
 	    <p>Milestone:<br> {{ $input[0]['milestones'][0]['category'] }} - {{ $input[0]['milestones'][0]['detail'] }}</p>
 	    <p>Objective:<br> {{ $input[0]['objectives'] }}</p>
+		@if ($_REQUEST['prefAnest1'] != "No Preference")  <!-- if an anesthesiologist was chosen, print their name, if not then print No Preference--> 
+		<p>Preferred Anesthesiologist:<br> {{$input[0]['prefAnest'][0]['first_name'] }} {{ $input[0]['prefAnest'][0]['last_name']}}</p>
+			@else
+			<p> Preferred Anesthesiologist:<br> {{ $_REQUEST['prefAnest1']}} </p>
```

Comment:
In this branch, won't it always say "No Preference"? I think it's clearer if we replace it with: `<p> Preferred Anesthesiologist:<br>No Preference</p>`